about.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+  <title>The role of technology in tutoring</title>
+</head>
+
+<body class="body">
+  <section class="landing-page">
+    <div class="top-header-div">
+      <ul class="top-header">
+        <li><a href="#" class="facebook"><img src="./images/icons8-facebook-f.svg" alt="This is a twitter button"></a></li>
+        <li><a href="#" class="twitter"><img src="./images/twitter.svg" alt="This is a twitter button"></a></li>
+        <li><a href="#" class="top-header-items">English</li>
+        <li><a href="#" class="top-header-items">My Page</li>
+        <li><a href="#" class="top-header-items">Logout</li>
+      </ul>
+    </div>
+    <header class="header-container">
+      <a href="index.html" class="my-logo"><img src="./images/logo.png" alt="This is my Portfolio Logo"></a>
+      <nav class="navigation">
+        <a href="" class="my-menu" title="menu bar"></a>
+        <ul class="desktop-nav">
+          <li class="nav-men"><a href="about.html" class="nav-link">About</a></li>
+          <li class="nav-men"><a href="#" class="nav-link">Program</a></li>
+          <li class="nav-men"><a href="#" class="nav-link">Join</a></li>
+          <li class="nav-men"><a href="#" class="nav-link">Sponsor</a></li>
+          <li class="nav-men"><a href="#" class="nav-link">News</a></li>
+          <li class="nav-men-cc"><a href="#" class="nav-link-cc">CC Campaign</a></li>
+        </ul>
+        <div id="mobilenav" class="mobileNavigation">
+          <a href="javascript:void(0)" class="killnav" onclick="menukill()">&times;</a>
+          <a href="index.html" class="close-menu">Home</a>
+          <a href="about.html" class="close-menu">About</a>
+          <a href="#" class="close-menu">Program</a>
+          <a href="#" class="close-menu">Join</a>
+          <a href="#" class="close-menu">Sponsor</a>
+          <a href="#" class="close-men">News</a>
+          <a href="#" class="close-men">CC Program</a>
+        </div>
+        <span class="navonmo" id="menu-toggle"><img src="./images/Union (2).png" alt="menu nav bar" style="width: 40px; height: 40px";></span>
+      </nav>
+    </header>
+    <div class="landing-page-div">
+      <h2 class="homepage-h2">"Hello! Sharing world"</h2>
+      <h1 class="homepage-h1">Revolutionizing Tutoring Through Technology Summit 2023  <img class="lineonabt" src="images/line.png" alt="line"></h1>
+      <h1 class="homepage-h8">Revolutionizing Tutoring Through <br> Technology Summit 2023  <img class="lineonabt" src="images/line.png" alt="line"></h1>
```

Comment:
- [ ] As mentioned previously please make sure to only use one main heading (`h1`) on your page the `h1` is used by search engines to identify the main content of the page and so adding more than one `h1` tag in your page will reduce the search ranking of your website.
- I noticed you want to make the text a two-liner so instead of repeating the heading you can use a fixed width of around `800px` for the heading on the style of the desktop version, this eliminates the need for the `br` tag. 😉